app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -6,7 +6,7 @@
 <% content_for :card do %>
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
     <h1 class="h2"><%= title %></h1>
-
+  <p><%= t('.instructions_html') %></p>
```

Comment:
[pebble] This will nest a `<p>` within a `<p>` because the content also contains a p. It should only be one or the other